src/app/views/landing/landing.component.html

```diff
@@ -9,7 +9,7 @@
       <div class="text-3xl text-primary font-bold mb-7">Panel kursanta</div>
       <p class="mt-0 mb-4 text-700 line-height-3">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
 
-      <button pButton pRipple label="Zaloguj" type="button" class="mr-3 p-button-raised"></button>
+      <button pButton pRipple label="Zaloguj" type="button" class="mr-3 p-button-raised" [routerLink]="'/login'"></button>
```

Comment:
To raczej nie będzie potrzebne bo logowanie jest załatwione przez oauth, przynajmniej na razie

Reply:
chyba że jednak router link beðzie potrzebny, ale przekieruje na home a nie na login, jeszcze tego nie sprawdziłem bo czekałem aż wrzucisz świeży kod :D

Reply:
spoko, to tak na potrzeby chwilowego przeklikania, mogę to wywalić żeby Wam nie śmieciło w repo jak tego nie chcecie ;)

Reply:
spoko ja to zmienie według tego co próbuję zaimplementować